probot/server/presolver.js

```diff
@@ -1,23 +1,26 @@
+const { updateDb } = require('./tools/update-db');
+
 class Presolver {
   constructor(context, { owner, repo, logger = console, ...config }) {
     this.context = context;
     this.github = context.github;
     this.logger = logger;
     this.config = {
-      ...require('./defaults'),
+      ...require('../../lib/defaults'),
       ...(config || {}),
       ...{
         owner,
         repo
       }
     };
-    // console.log(this.config)
     this.pullRequest = {};
     this.conflictingFiles = [];
+		this._updateDb = updateDb;
   }
 
   async presolve(pullRequest) {
     Object.assign(this.pullRequest, pullRequest);
+    // await this._updateDb(pullRequest);
```

Comment:
@RandellDawson I can't get this running due to lib/get-prs getFilenames requiring credentials, but maybe it will work on your end.